### PR TITLE
Decision doc

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Please find attached a decision document initial submission--feedback and suggested additions are welcome!

**Why I am suggesting this:** In discussion on https://reviewable.io/reviews/datacommonsorg/docsite/97, it became clear that some sort of standard was needed for when to describe (for example) objects as places or `Place` objects. After thinking about, it seemed to me that having a separate decision document for contributors to refer to might be a good idea--for example, if we note decisions in the readme, it might become cluttered.

### CODE REVIEW

**Review by beets — APPROVED** [review]
This is great, thanks for starting this doc.

A couple of suggestions:
- renaming the file STYLE_GUIDE.md
- Referencing it from the README

**Comment by KilimAnnejaro** [comment]
updated!

File: README.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ git push -u origin new_branch_name
 
 Then in your forked repo, you can send a Pull Request. If this is your first
 time contributing to a Google Open Source project, you may need to follow the
-steps in [contributing.md](contributing.md).
+steps in [contributing.md](contributing.md). Be sure to follow STYLE_GUIDE.md
```
Comment (tjann) [inline README.md:59]: For consistency, do [STYLE_GUIDE.md](STYLE_GUIDE.md) like for contributing.md?

**Review by tjann — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (beets) [reply README.md:59]: I think Tiffany was suggesting adding a link to the style guide here.

**Review by beets — CHANGES_REQUESTED** [review]
great thanks for the update! one small clarification on a comment.

**Review by beets — APPROVED** [review]

**Pull request merged**